Cilsil.Test/Assets/Utils.cs

```diff
@@ -57,7 +57,8 @@ public enum InferError
             NULL_DEREFERENCE,
             DANGLING_POINTER_DEREFERENCE,
             DOTNET_RESOURCE_LEAK,
-            THREAD_SAFETY_VIOLATION
```

Comment:
you need to add support for this at line 192 in getstring:

![image](https://user-images.githubusercontent.com/38902361/149399321-37557f27-7a05-4146-a6cf-8ca883022170.png)